1.8/usage/tutorials/tutorial/resources.md

```diff
@@ -0,0 +1,99 @@
+---
+post_title: Understanding Ressources
+nav_title: Understanding Ressources
+menu_order: 8
+---
+
+# Prerequisites
+* A [running DC/OS cluster](/docs/1.8/tutorial/cli/) with [the DC/OS CLI installed](link to installation instructions].
+* [app2](/docs/1.8/tutorial/app2/) deployed and running in your cluster.
+
+# Objective
+In this section we will learn how to monitor and understand our resource utilization, how resource limits are enforced, and how to debug resource management issues.
+
+# Steps
+Resource management and resource isolation between tasks are core challenges for an operating system, which needs to manage resources.
+
+* Deploy App
+  * Let us have a second look at the defintion of [app2](https://github.com/joerg84/dcos-101/blob/master/app2/app2.go).
+
```

Comment:
Yes, probably. :-)

Also, I just noticed "Resources" is spelled wrong in the post and nav title.